src/sidebar/components/test/selection-tabs-test.js

```diff
@@ -69,14 +53,23 @@ describe('SelectionTabs', function() {
         },
       }),
     };
+
+    SelectionTabs.$imports.$mock(mockImportedComponents());
+    SelectionTabs.$imports.$mock({
```

Comment:
could mockImportedComponents be modified to take on overrides so we don't have to make multiple calls using 
`SomeComponent.$imports.$mock(...)`

Reply:
That would certainly be possible, although it may end up re-implementing some of `$mock`'s logic. Alternatively `$mock` could be changed to accept multiple arguments or an array so you could use it something like this:

```js
SelectionTabs.$imports.$mock(
  mockImportedComponents(),
  {
    // Other imports here.
  }
)
```